drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/model/DropInPaymentMethodInformation.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ararat on 9/11/2023.
+ */
+
+package com.adyen.checkout.dropin.internal.ui.model
+
+import android.os.Parcelable
+import com.adyen.checkout.components.core.PaymentMethod
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class DropInPaymentMethodInformation(
```

Comment:
Can this class be internal?

Reply:
Yes, it will be fixed with the next commit